beams/beams/custom_scripts/job_applicant/job_applicant.py

```diff
@@ -30,16 +30,33 @@ def validate(doc, method):
     Method triggered before the document is saved.
     Validate the Job Applicant against Job Opening requirements.
     """
-    if not frappe.db.exists("Job Opening", {"name": doc.job_title}):
-        frappe.throw(_("The specified Job Opening does not exist."))
```

Comment:
if job_title is not reqd, adjust the corresponding validations to work only when it is filled in